gtsam/base/utilities.cpp

```diff
@@ -0,0 +1,13 @@
+#include <gtsam/base/utilities.h>
+
+namespace gtsam {
+
+std::string RedirectCout::str() const {
+  return ssBuffer_.str();
+}
+
+RedirectCout::~RedirectCout() {
+  std::cout.rdbuf(coutBuffer_);
+}
+
+}
```

Comment:
FYI if you run the formatter on this file, it should add the `// namespace gtsam` comment at this point.